frontend/package.json

```diff
@@ -44,7 +44,7 @@
     "react-test-renderer": "^17.0.2"
   },
   "scripts": {
-    "start": "env-cmd -f ../.env -e development react-scripts start",
+    "start": "BROWSER=none env-cmd -f ../.env -e development react-scripts start",
```

Comment:
Didn't know about this trick, thanks for finding it!

Reply:
Of course! I was quite annoyed by the incessant `localhost:3000` popups :)

Reply:
This is helpful I've been wondering how you took out the `localhost:3000` popups!

Reply:
Oh, wow, this is super nice!  Thanks for finding this.  We'll incorporate this into our future apps (much easier than the code that makes the special warning pop up when you are on port 3000).

Reply:
No problem, @pconrad! Glad to hear it will be of use to you in the future!